theories/.dir-locals.el

```diff
@@ -0,0 +1,6 @@
+((coq-mode . ((eval . (let ((default-directory (locate-dominating-file
+						buffer-file-name ".dir-locals.el")))
+			(make-local-variable 'coq-prog-args)
+			(make-local-variable 'coq-prog-name)
+			(setq coq-prog-args `("-emacs" "-coqlib" ,(expand-file-name "..") "-R" ,(expand-file-name ".") "Coq"))
```

Comment:
This is true.  However, the system-wide Coq will almost certainly not work; I got interface mismatches on things like `nat_syntax_plugin`.  So users that want to edit the std lib currently have to manually set the coq binary.  This change fixes that problem for the common case of the default directory structure, and I'm not sure it's worth the overhead to have `./configure` autogenerate `.dir-locals.el`.  What do others think?